trigger/payloads.go

```diff
@@ -438,7 +438,7 @@ func getFailedTestsFromTemplate(t task.Task) ([]task.TestResult, error) {
 }
 
 func taskLink(uiBase string, taskID string, execution int) string {
-	return fmt.Sprintf("%s/task/%s/%d", uiBase, url.PathEscape(taskID), execution)
```

Comment:
should this consider hasPatch @khelif96 ?

Reply:
I don't think so since mainline commits are now supported on spruce we don't need to check if a patch exists.